kubernetes/redis/statefulset-sentinel.yaml

```diff
@@ -27,7 +27,7 @@ spec:
             do
                 for i in ${nodes//,/ }
                 do
-                    MASTER=$(redis-cli --no-auth-warning --raw -h $i --user admin -a $REDIS_PASSWORD info replication | awk '{print $1}' | grep master_host: | cut -d ":" -f2)
+                    MASTER=$(redis-cli --tls --cacert /certs/ca.crt --no-auth-warning --raw -h $i --user admin -a $REDIS_PASSWORD info replication | awk '{print $1}' | grep master_host: | cut -d ":" -f2)
```

Comment:
issue will probably crop up here if forcing x509, which can deal with when can test in future